packages/cozy-client/src/models/paper.js

```diff
@@ -7,6 +7,11 @@ import sub from 'date-fns/sub'
 
 const PERSONAL_SPORTING_LICENCE_PERIOD_DAYS = 365
 const PERSONAL_SPORTING_LICENCE_NOTICE_PERIOD_DAYS = 15
+const EXPIRATION_LINK_BY_LABEL = {
+  national_id_card: 'https://www.service-public.fr/particuliers/vosdroits/N358',
+  residence_permit: 'https://www.service-public.fr/particuliers/vosdroits/N110',
+  passport: 'https://www.service-public.fr/particuliers/vosdroits/N360'
+}
```

Comment:
Shouldn't we add some localization? If I am spanish, I don't expect to be redirected to the french passport page

Reply:
C'est envisageable, mais sans partir dans divers sites dont j'ignore l'existence, ce dernier ne propose que les langues `fr` et `en`(par traduction automatique) via un query param `lang=[fr|en]`.

On pourrait donc rediriger toutes les langues sur la version anglaise ? 🤷

Reply:
Après discussion avec le PO, des solutions sont en réflexions, mais laissons en l'état pour le moment

Reply:
> On pourrait donc rediriger toutes les langues (autres que FR) sur la version anglaise en se basant sur l'attribut country ? shrug (ou ne pas mettre de lien)

Pour moi la logique serait de proposer ces liens si country=fr, et ne rien proposer sinon

Reply:
Aujourd'hui, aucuns papiers ne défini la metadata `country` à `fr`.
Pour le moment, seul la création d'un Permis(Etranger) renseigne cette metadata (à la valeur définie par l'utilisateur).

De ce fait, on ne peut pas être certain aujourd'hui qu'un autre papier est FR

Reply:
> Aujourd'hui, aucuns papiers ne défini la metadata country à fr

c'est peut être le coeur du problème :smile: Il y a une raison à ça ? on devrait le généraliser, non ?

Reply:
car de base on en avait pas besoin c'est suite à la distinction entre Permis FR / Permis Etranger que cette variable à commencer à prendre du sens mais oui comme échangé sur zoom je pense qu'il faudrait généraliser.

Reply:
Ok, vu avec @BenjaminMty, une carte arrivera prochainement à propos de ce sujet!